<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-11</x>
      <y>-4</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>779</width>
       <height>389</height>
      </rect>
     </property>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>681</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar_como"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuEdici_n">
    <property name="title">
     <string>Edici√≥n</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionSeleccionar_todo"/>
   </widget>
   <widget class="QMenu" name="menuHerraminetas">
    <property name="title">
     <string>Herraminetas</string>
    </property>
    <addaction name="actionNuevo"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdici_n"/>
   <addaction name="menuHerraminetas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbrir"/>
   <addaction name="actionNuevo"/>
   <addaction name="separator"/>
   <addaction name="actionCopiar"/>
   <addaction name="actionCortar"/>
   <addaction name="actionPegar"/>
   <addaction name="actionSeleccionar_todo"/>
   <addaction name="separator"/>
   <addaction name="actionAcerca_de"/>
   <addaction name="actionCerrar"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Delete.ico</normaloff>:/iconos/Delete.ico</iconset>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Add Folder.ico</normaloff>:/iconos/Add Folder.ico</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Stop 2.ico</normaloff>:/iconos/Stop 2.ico</iconset>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Copy.ico</normaloff>:/iconos/Copy.ico</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Fonts 2.ico</normaloff>:/iconos/Fonts 2.ico</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionSeleccionar_todo">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Spell.ico</normaloff>:/iconos/Spell.ico</iconset>
   </property>
   <property name="text">
    <string>Seleccionar todo</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Cut.ico</normaloff>:/iconos/Cut.ico</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Chat 2.ico</normaloff>:/iconos/Chat 2.ico</iconset>
   </property>
   <property name="text">
    <string>Acerca de..</string>
   </property>
  </action>
  <action name="actionGuardar_como">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/iconos/Save.ico</normaloff>:/iconos/Save.ico</iconset>
   </property>
   <property name="text">
    <string>Guardar como..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCerrar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
